tests/test_api_auth.py

```diff
@@ -0,0 +1,356 @@
+# -*- coding: utf-8 -*-
+""" Test for the '/auth' API-endpoint """
+
+from .base import MyApiTestCase
+import mock
+import six
+from privacyidea.lib.config import set_privacyidea_config
+
+
+class AuthApiTestCase(MyApiTestCase):
```

Comment:
Could you please add a test case for an internal admin?
We have this one "testadmin"/"testpw", but in theory you could also define an internal admin like "admin@int", even if the "int" realm does not exist. And I think the code will and should work, since we do not raise an error (which is fine) the pseudo realm "int" would be split and the admin is verify with the complete ``username`` "admin@int".

Reply:
will do

Reply:
Done in 3bcd19af